### ISSUE TITLE
Problem creating new accounts

### ISSUE THREAD

**Issue by Alex99y** [cosmos/sdk-tutorials#668]
Hello, playing with the scripts I was able to create new accounts using the next command:
`nameservicecli keys add newAccount`
But now, after resetting I get this error when I want to create or recover any account:
```sh
panic: crypto/hmac: hash generation function does not produce unique values

goroutine 1 [running]:
crypto/hmac.New(0xc0001f4ea8, 0xc001107630, 0x4, 0x8, 0x0, 0x8)
	/usr/local/go/src/crypto/hmac/hmac.go:143 +0x370
github.com/dvsekhvalnov/jose2go/kdf.DerivePBKDF2(0xc001107630, 0x4, 0x8, 0xc0010defc0, 0x1f, 0x30, 0x2000, 0x80, 0x13d2ab8, 0xc0002b6980, ...)
	/home/alex99y/go/pkg/mod/github.com/dvsekhvalnov/jose2go@v0.0.0-20180829124132-7f401d37b68a/kdf/pbkdf2.go:15 +0xad
github.com/dvsekhvalnov/jose2go.(*Pbse2HmacAesKW).WrapNewKey(0xc000824c30, 0x100, 0xf289a0, 0xc00113e290, 0xc001109a40, 0x203001, 0x40, 0x40, 0x3, 0x34000000004c570a, ...)
	/home/alex99y/go/pkg/mod/github.com/dvsekhvalnov/jose2go@v0.0.0-20180829124132-7f401d37b68a/pbse2_hmac_aeskw.go:50 +0x439
github.com/dvsekhvalnov/jose2go.encrypt(0xc00114a360, 0x107, 0x120, 0xc001109a40, 0xf289a0, 0xc00113e290, 0xc00114a467, 0x19, 0x0, 0xc00114a360)
	/home/alex99y/go/pkg/mod/github.com/dvsekhvalnov/jose2go@v0.0.0-20180829124132-7f401d37b68a/jose.go:305 +0x20a
github.com/dvsekhvalnov/jose2go.EncryptBytes(0xc00114a360, 0x107, 0x120, 0x10bfad6, 0x12, 0x10b4fb9, 0x7, 0xf289a0, 0xc00113e290, 0xc0001f53d0, ...)
	/home/alex99y/go/pkg/mod/github.com/dvsekhvalnov/jose2go@v0.0.0-20180829124132-7f401d37b68a/jose.go:222 +0x32f
github.com/dvsekhvalnov/jose2go.Encrypt(...)
	/home/alex99y/go/pkg/mod/github.com/dvsekhvalnov/jose2go@v0.0.0-20180829124132-7f401d37b68a/jose.go:187
github.com/99designs/keyring.(*fileKeyring).Set(0xc001109710, 0xc001107490, 0x9, 0xc0002bc2d0, 0x65, 0xe4, 0x0, 0x0, 0x0, 0x0, ...)
	/home/alex99y/go/pkg/mod/github.com/99designs/keyring@v1.1.3/file.go:143 +0x319
github.com/cosmos/cosmos-sdk/crypto/keys.keyringKeybase.writeInfo(0x12b2638, 0x12b2630, 0xc0010fff80, 0x1, 0x1, 0xc0010fff90, 0x1, 0x1, 0x13d2bf8, 0xc001109710, ...)
	/home/alex99y/go/pkg/mod/github.com/cosmos/cosmos-sdk@v0.39.1/crypto/keys/keyring.go:473 +0x218
github.com/cosmos/cosmos-sdk/crypto/keys.keyringKeybase.writeLocalKey(0x12b2638, 0x12b2630, 0xc0010fff80, 0x1, 0x1, 0xc0010fff90, 0x1, 0x1, 0x13d2bf8, 0xc001109710, ...)
	/home/alex99y/go/pkg/mod/github.com/cosmos/cosmos-sdk@v0.39.1/crypto/keys/keyring.go:464 +0x193
github.com/cosmos/cosmos-sdk/crypto/keys.baseKeybase.CreateAccount(0x12b2638, 0x12b2630, 0xc0010fff80, 0x1, 0x1, 0xc0010fff90, 0x1, 0x1, 0x13c45f0, 0xc0002d4410, ...)
	/home/alex99y/go/pkg/mod/github.com/cosmos/cosmos-sdk@v0.39.1/crypto/keys/keybase_base.go:171 +0x1a2
github.com/cosmos/cosmos-sdk/crypto/keys.keyringKeybase.CreateAccount(...)
	/home/alex99y/go/pkg/mod/github.com/cosmos/cosmos-sdk@v0.39.1/crypto/keys/keyring.go:106
github.com/cosmos/cosmos-sdk/client/keys.RunAddCmd(0xc001123400, 0xc0010f8880, 0x1, 0x2, 0x13e03d8, 0xc0002d43c0, 0xc00112e3c0, 0xc0001f5c40, 0x5d24c5)
	/home/alex99y/go/pkg/mod/github.com/cosmos/cosmos-sdk@v0.39.1/client/keys/add.go:273 +0x743
github.com/cosmos/cosmos-sdk/client/keys.runAddCmd(0xc001123400, 0xc0010f8880, 0x1, 0x2, 0x0, 0x0)
	/home/alex99y/go/pkg/mod/github.com/cosmos/cosmos-sdk@v0.39.1/client/keys/add.go:99 +0x170
github.com/spf13/cobra.(*Command).execute(0xc001123400, 0xc0010f8860, 0x2, 0x2, 0xc001123400, 0xc0010f8860)
	/home/alex99y/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:852 +0x472
github.com/spf13/cobra.(*Command).ExecuteC(0xc0010ee000, 0xc0001f5e68, 0x863485, 0xc0010ee000)
	/home/alex99y/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:960 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/home/alex99y/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:897
github.com/tendermint/tendermint/libs/cli.Executor.Execute(0xc0010ee000, 0x12b5070, 0x2, 0xc0010d72a0)
	/home/alex99y/go/pkg/mod/github.com/tendermint/tendermint@v0.33.7/libs/cli/setup.go:89 +0x3c
main.main()
	/home/alex99y/Escritorio/sdk-tutorials/nameservice/nameservice/cmd/nameservicecli/main.go:72 +0x34e
```

**Comment by toschdev** [comment]
Hello @Alex99y, which version of Golang are you using? Please check with `go version` and post here.
Each tutorial should state a recommended go version.

**Comment by Alex99y** [comment]
@toschdev thanks for answering me. 
go version output: `go version go1.16.4 linux/amd64`

**Comment by toschdev** [comment]
The `nameservice` tutorial uses go version 1.15 

When you downgrade for this tutorial it should be working 👍

**Comment by Alex99y** [comment]
@toschdev  works perfectly! Thank you a ton!

**Comment by assafmo** [comment]
Happened to me as well! I can confirm that downgrading Go solved the issue (Tried with 1.17.1, downgraded down to 1.13.8)
